<?xml version="1.0" ?>
<launch>
  <arg name="odometryTopic" default="/curiosity_mars_rover/odom" />
  <arg name="baseFrame" default="base_footprint" />
  <arg name="odomFrame" default="world" />

  <node name="odom_to_tf" pkg="odom_to_tf" type="odom_to_tf" output="screen">
    <param name="odometryTopic" value="$(arg odometryTopic)" />
    <param name="baseFrame" value="$(arg baseFrame)" />
    <param name="odomFrame" value="$(arg odomFrame)" />
  </node>
</launch>
